content/posts/newsletter/2018-05.md

```diff
@@ -0,0 +1,123 @@
+---
+title: The XMPP Newsletter, 1 June 2018
+date: 2018-05-03 00:00
+author: jcbrand
+blog_id: newsletter 
+category: Newsletter
+---
+
+Welcome to another edition of the XMPP newsletter.
+
+## News
+
+The next release of [Salut à Toi will include file sharing](https://www.goffi.org/b/hGKs6B4wd8dsgNZd5MzQjN/file-sharing-landing-next-release-salut?utm_source=xmppnewsletter).
+Sharing files directly between two users has been possible before, but now it's
+possible to share a file hierarchy, or in other words one or several
+directories. To share with someone, just use their XMPP address (JID).
+
+[Christopher Muclumbus](https://muclumbus.jabbercat.org/?utm_source=xmppnewsletter) 
```

Comment:
You can remove the tracking here, I don’t make use of it (and I would prefer if it didn’t look as if I did).

Reply:
I'll remove it, but the fact that it's there doesn't imply that you use tracking, it's added by the newsletter, not the site owner.